content/docs/nav.yml

```
@@ -1,153 +1,154 @@
 - title: التنصيب
   items:
     - id: getting-started
-      title: Getting Started
+      title: البدء مع React
     - id: add-react-to-a-website
-      title: Add React to a Website
+      title: إضافة React للموقع
```

<div dir="rtl">
في النص الأصلي كلمة "موقع" غير معرّفة، أقترح العبارة التالية:
</div>

```suggestion
      title: إضافة React لموقع
```

<div dir="rtl">
أو (العبارة المستعملة في موسوعة <a href="https://wiki.hsoub.com/React#.D8.A5.D8.B6.D8.A7.D9.81.D8.A9_React_.D8.A5.D9.84.D9.89_.D9.85.D9.88.D9.82.D8.B9_.D9.88.D9.8A.D8.A8">حسوب</a> )
</div>

```suggestion
      title: إضافة React إلى موقع ويب
```

---

اولا شكرا لك على مجهودك اخي الكريم .. 

انا ارى ان الموقع معرف لدى الزائر لانه سيقوم بإظافة عناصر REACT JS إلى موقع هو صاحبه او يعمل عليه و بذالك تكون معرفة لدى الزائر .. ان كان هناك راي اخر لا باس سنقوم بالتعديل لا مشكلة ..